### ISSUE TITLE
Headless blacklist value added when opening & closing unit attributes - causes RPT spam

### ISSUE THREAD

**Issue by Kresky** [acemod/ACEX#120]
**Arma 3 Version:** `1.8.143869` (stable)
**CBA Version:** `3.6.1` (stable)
**ACE3 Version:** `3.12.1` (stable)
**ACEX Version:** `3.3.0` (stable)

**Mods:**
```
- CBA_A3
- ace
- acex
```

**Description:**
- Opening and closing unit attributes in Eden causes the following to get added in the misson.sqm:
```[...]
type="B_Soldier_SL_F";
atlOffset=1.5258789e-005;
class CustomAttributes
{
	class Attribute0
	{
		property="acex_headless_blacklist";
		expression="_this setVariable [""acex_headless_blacklist"",_value,true]";
		class Value
		{
			class data
			{
				class type
				{
					type[]=
					{
						"BOOL"
					};
				};
				value=0;
			};
		};
	};
[...]
```

- This seems to cause the following to be spammed in RPT on dedicated server:
```
if (_x getVariable ["acex_headless_black>
14:00:50   Error position: <if (_x getVariable ["acex_headless_black>
14:00:50   Error if: Type Number, expected Bool
14:00:50 File z\acex\addons\headless\functions\fnc_transferGroups.sqf, line 74
14:00:50 [ACEX] (headless) INFO: Present HCs: [headlessclient1,headlessclient2,headlessclient3] - Full Rebalance: true
14:00:50 Error in expression < _x) exitWith {
_transfer = false;
};
```

**Steps to reproduce:**
- Place down unit in Eden editor
- Open attributes, then close again (no need to change anything)

**Where did the issue occur?**
- Eden Editor

**Placed Modules:**
- None

**Comment by jonpas** [comment]
Shut up bot! :laughing:

**Comment by jonpas** [comment]
Fixed by #141 right @PabstMirror ?
